wcfsetup/install/files/lib/acp/form/BoxAddForm.class.php

```diff
@@ -393,6 +393,37 @@ public function readFormParameters()
         if ($this->boxType === 'system') {
             $this->boxController = ObjectTypeCache::getInstance()->getObjectType($this->boxControllerID);
         }
+
+        $this->readConditions();
+    }
+
+    private function readConditions(): void
+    {
+        foreach ($this->groupedConditionObjectTypes as $groupedObjectTypes) {
+            foreach ($groupedObjectTypes as $objectTypes) {
+                if (\is_array($objectTypes)) {
```

Comment:
In which cases is this not an array? Can you avoid the duplication by doing the following:

```
if (!\is_array($objectTypes)) {
  $objectTypes = [$objectTypes];
}
```
?